dotCMS/src/main/java/com/dotcms/rest/api/v1/folder/FolderResource.java

```diff
@@ -106,4 +116,216 @@ public final Response loadFolderByURI(@Context final HttpServletRequest httpServ
         return response;
     }
 
+    @GET
+    @Path ("/sitename/{siteName}/folder/{folder : .+}")
```

Comment:
I might be wrong, but shouldn't this be something like: `/site/${siteName}/folder/{folder : .+}`

Reply:
I agree but I tried to keep the same parameter name than we had in a previous endpoint.

Reply:
ok nvm